### PR TITLE
【Prim】Fix slice error and eager comp

### PR DESCRIPTION
**Pull request by JiabinYang**
### PR types

Bug fixes
### PR changes

Others
### Describe

This PR did three things
1. Fix slice error when using decrease axis
2. Hide additional log info by flag `FLAGS_prim_log`
3. Add `set_prim_eager_enabled` to control eager composition independently

### CODE REVIEW

**Review by zyfncg — APPROVED** [review]
LGTM for eager_gen.py

File: paddle/fluid/prim/utils/utils.cc [hunk 1]
```diff
@@ -27,6 +27,14 @@ void PrimCommonUtils::SetBwdPrimEnabled(bool enable_prim) {
...
 }
 
+bool PrimCommonUtils::IsEagerPrimEnabled() {
+  return StaticCompositeContext::Instance().IsEagerPrimEnabled();
+}
+
+void PrimCommonUtils::SetEagerPrimEnabled(bool enable_prim) {
+  StaticCompositeContext::Instance().SetEagerPrimEnabled(enable_prim);
+}
+
```
Comment (xiaoguoguo626807) [inline paddle/fluid/prim/utils/utils.cc:37]: it's better to add some describe for these flag

File: python/paddle/fluid/core.py [hunk 2]
```diff
@@ -316,6 +316,8 @@ def to_list(s):
     from .libpaddle import __set_fwd_prim_enabled
     from .libpaddle import _is_fwd_prim_enabled
     from .libpaddle import __set_all_prim_enabled
+    from .libpaddle import _is_eager_prim_enabled
+    from .libpaddle import __set_eager_prim_enabled
```
Comment (xiaoguoguo626807) [inline python/paddle/fluid/core.py:320]: why set use two _ ?

**Review by xiaoguoguo626807 — APPROVED** [review]

**Review by zhiqiu — APPROVED** [review]
LGTM for test_slice

File: python/paddle/fluid/core.py (hunk 2 above)
Comment (JiabinYang) [reply python/paddle/fluid/core.py:320]: in this way, no one can directly use it, out of core

File: paddle/fluid/prim/utils/utils.cc (hunk 1 above)
Comment (JiabinYang) [reply paddle/fluid/prim/utils/utils.cc:37]: Sure, i can do this in next pr

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Pull request merged**